1.4.10 quickstart-se fails to build native-image with GraalVM 21.3.0

## Environment Details
* Helidon Version: 1.4.10
* Helidon SE 
* JDK version: 11.0.13 2021-10-19 LTS
* native-image version: GraalVM 21.3.0 Java 11 CE (Java Version 11.0.13+7-jvmci-21.3-b05)
* OS: MacOS

----------

## Problem Description
Building the 1.4.10 quickstart-se fails to build native-image with GraalVM 21.3.0 with this error:
```
[INFO] [helidon-quickstart-se:28824]     analysis:  20,190.57 ms,  3.80 GB
[WARNING] Error: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: 
Detected an instance of Random/SplittableRandom class in the image heap. 
Instances created during image generation have cached seed values and don't behave as expected.  
To see how this object got instantiated use --trace-object-instantiation=java.util.Random. 
The object was probably created by a class initializer and is reachable from a static field. 
You can request class initialization at image runtime by using the option --initialize-at-run-time=<class-name>. 
Or you can write your own initialization methods and call them explicitly from your main entry point.
[WARNING] Detailed message:
[WARNING] Trace:
[WARNING] 	at parsing io.helidon.webserver.SSLContextBuilder.buildKmf(SSLContextBuilder.java:169)
[WARNING] Call path from entry point to io.helidon.webserver.SSLContextBuilder.build():
[WARNING] 	at io.helidon.webserver.SSLContextBuilder.build(SSLContextBuilder.java:131)
[WARNING] 	at io.helidon.webserver.SSLContextBuilder.build(SSLContextBuilder.java:44)
[WARNING] 	at io.helidon.common.Builder.get(Builder.java:41)
[WARNING] 	at com.oracle.svm.core.jdk.SystemPropertiesSupport.initializeLazyValue(SystemPropertiesSupport.java:216)
[WARNING] 	at com.oracle.svm.core.jdk.SystemPropertiesSupport.getProperty(SystemPropertiesSupport.java:169)
[WARNING] 	at com.oracle.svm.core.jdk.Target_java_lang_System.getProperty(JavaLangSubstitutions.java:287)
[WARNING] 	at com.oracle.svm.jni.JNIJavaCallWrappers.jniInvoke_ARRAY_System_getProperty_deeeaa72a006d330408a3b7d002c7533e108bc28(generated:0)
[WARNING]
[WARNING] com.oracle.svm.core.util.UserError$UserException: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Detected an instance of Random/SplittableRandom class in the image heap. Instances created during image generation have cached seed values and don't behave as expected.  To see how this object got instantiated use --trace-object-instantiation=java.util.Random. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image runtime by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
[WARNING] Detailed message:
[WARNING] Trace:
[WARNING] 	at parsing io.helidon.webserver.SSLContextBuilder.buildKmf(SSLContextBuilder.java:169)
[WARNING] Call path from entry point to io.helidon.webserver.SSLContextBuilder.build():
[WARNING] 	at io.helidon.webserver.SSLContextBuilder.build(SSLContextBuilder.java:131)
[WARNING] 	at io.helidon.webserver.SSLContextBuilder.build(SSLContextBuilder.java:44)
[WARNING] 	at io.helidon.common.Builder.get(Builder.java:41)
[WARNING] 	at com.oracle.svm.core.jdk.SystemPropertiesSupport.initializeLazyValue(SystemPropertiesSupport.java:216)
[WARNING] 	at com.oracle.svm.core.jdk.SystemPropertiesSupport.getProperty(SystemPropertiesSupport.java:169)
[WARNING] 	at com.oracle.svm.core.jdk.Target_java_lang_System.getProperty(JavaLangSubstitutions.java:287)
[WARNING] 	at com.oracle.svm.jni.JNIJavaCallWrappers.jniInvoke_ARRAY_System_getProperty_deeeaa72a006d330408a3b7d002c7533e108bc28(generated:0)
[WARNING]
[WARNING] 	at com.oracle.svm.core.util.UserError.abort(UserError.java:87)
[WARNING] 	at com.oracle.svm.hosted.FallbackFeature.reportAsFallback(FallbackFeature.java:233)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:759)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:529)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:488)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:403)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:569)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:122)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGeneratorRunner$JDK9Plus.main(NativeImageGeneratorRunner.java:599)
[WARNING] Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Detected an instance of Random/SplittableRandom class in the image heap. Instances created during image generation have cached seed values and don't behave as expected.  To see how this object got instantiated use --trace-object-instantiation=java.util.Random. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image runtime by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
[WARNING] Detailed message:
[WARNING] Trace:
[WARNING] 	at parsing io.helidon.webserver.SSLContextBuilder.buildKmf(SSLContextBuilder.java:169)
[WARNING] Call path from entry point to io.helidon.webserver.SSLContextBuilder.build():
[WARNING] 	at io.helidon.webserver.SSLContextBuilder.build(SSLContextBuilder.java:131)
[WARNING] 	at io.helidon.webserver.SSLContextBuilder.build(SSLContextBuilder.java:44)
[WARNING] 	at io.helidon.common.Builder.get(Builder.java:41)
[WARNING] 	at com.oracle.svm.core.jdk.SystemPropertiesSupport.initializeLazyValue(SystemPropertiesSupport.java:216)
[WARNING] 	at com.oracle.svm.core.jdk.SystemPropertiesSupport.getProperty(SystemPropertiesSupport.java:169)
[WARNING] 	at com.oracle.svm.core.jdk.Target_java_lang_System.getProperty(JavaLangSubstitutions.java:287)
[WARNING] 	at com.oracle.svm.jni.JNIJavaCallWrappers.jniInvoke_ARRAY_System_getProperty_deeeaa72a006d330408a3b7d002c7533e108bc28(generated:0)
[WARNING]
[WARNING] 	at com.oracle.graal.pointsto.constraints.UnsupportedFeatures.report(UnsupportedFeatures.java:126)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:756)
[WARNING] 	... 6 more
[WARNING] Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Detected an instance of Random/SplittableRandom class in the image heap. Instances created during image generation have cached seed values and don't behave as expected.  To see how this object got instantiated use --trace-object-instantiation=java.util.Random. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image runtime by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
[WARNING] 	at com.oracle.svm.hosted.image.DisallowedImageHeapObjectFeature.error(DisallowedImageHeapObjectFeature.java:173)
[INFO] [helidon-quickstart-se:28824]      [total]:  25,483.98 ms,  3.80 GB
[WARNING] 	at com.oracle.svm.core.image.DisallowedImageHeapObjects.check(DisallowedImageHeapObjects.java:65)
[WARNING] 	at com.oracle.svm.hosted.image.DisallowedImageHeapObjectFeature.replacer(DisallowedImageHeapObjectFeature.java:149)
[WARNING] 	at com.oracle.graal.pointsto.meta.AnalysisUniverse.replaceObject(AnalysisUniverse.java:570)
[WARNING] 	at com.oracle.svm.hosted.ameta.AnalysisConstantReflectionProvider.replaceObject(AnalysisConstantReflectionProvider.java:217)
[WARNING] 	at com.oracle.svm.hosted.ameta.AnalysisConstantReflectionProvider.interceptValue(AnalysisConstantReflectionProvider.java:188)
[WARNING] 	at com.oracle.svm.hosted.ameta.AnalysisConstantReflectionProvider.readValue(AnalysisConstantReflectionProvider.java:102)
[WARNING] 	at com.oracle.svm.hosted.ameta.AnalysisConstantReflectionProvider.readFieldValue(AnalysisConstantReflectionProvider.java:81)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.util.ConstantFoldUtil$1.readValue(ConstantFoldUtil.java:51)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.core.common.spi.JavaConstantFieldProvider.readConstantField(JavaConstantFieldProvider.java:84)
[WARNING] 	at com.oracle.svm.hosted.ameta.AnalysisConstantFieldProvider.readConstantField(AnalysisConstantFieldProvider.java:72)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.util.ConstantFoldUtil.tryConstantFold(ConstantFoldUtil.java:47)
[WARNING] 	at com.oracle.svm.hosted.phases.ConstantFoldLoadFieldPlugin.tryConstantFold(ConstantFoldLoadFieldPlugin.java:61)
[WARNING] 	at com.oracle.svm.hosted.phases.ConstantFoldLoadFieldPlugin.handleLoadStaticField(ConstantFoldLoadFieldPlugin.java:57)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genGetStatic(BytecodeParser.java:4944)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genGetStatic(BytecodeParser.java:4911)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5413)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3477)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.handleBytecodeBlock(BytecodeParser.java:3437)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3282)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1145)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:1030)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:84)
[WARNING] 	at com.oracle.svm.hosted.phases.SharedGraphBuilderPhase.run(SharedGraphBuilderPhase.java:81)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:212)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
[WARNING] 	at com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:132)
[WARNING] 	at com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:621)
[WARNING] 	at com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.lookupEncodedGraph(InlineBeforeAnalysis.java:182)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.doInline(PEGraphDecoder.java:1120)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.tryInline(PEGraphDecoder.java:1103)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.trySimplifyInvoke(PEGraphDecoder.java:961)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.handleInvoke(PEGraphDecoder.java:915)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.processNextNode(GraphDecoder.java:791)
[WARNING] 	at com.oracle.graal.pointsto.phases.InlineBeforeAnalysisGraphDecoder.processNextNode(InlineBeforeAnalysis.java:242)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.nodes.GraphDecoder.decode(GraphDecoder.java:532)
[WARNING] 	at jdk.internal.vm.compiler/org.graalvm.compiler.replacements.PEGraphDecoder.decode(PEGraphDecoder.java:787)
[WARNING] 	at com.oracle.graal.pointsto.phases.InlineBeforeAnalysis.decodeGraph(InlineBeforeAnalysis.java:99)
[WARNING] 	at com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:171)
[WARNING] 	at com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:321)
[WARNING] 	at com.oracle.graal.pointsto.flow.MethodTypeFlow.createTypeFlow(MethodTypeFlow.java:293)
[WARNING] 	at com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureTypeFlowCreated(MethodTypeFlow.java:282)
[WARNING] 	at com.oracle.graal.pointsto.flow.MethodTypeFlow.addContext(MethodTypeFlow.java:103)
[WARNING] 	at com.oracle.graal.pointsto.DefaultAnalysisPolicy$DefaultSpecialInvokeTypeFlow.onObservedUpdate(DefaultAnalysisPolicy.java:363)
[WARNING] 	at com.oracle.graal.pointsto.flow.TypeFlow.notifyObservers(TypeFlow.java:490)
[WARNING] 	at com.oracle.graal.pointsto.flow.TypeFlow.update(TypeFlow.java:559)
[WARNING] 	at com.oracle.graal.pointsto.PointsToAnalysis$2.run(PointsToAnalysis.java:595)
[WARNING] 	at com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:188)
[INFO] # Printing build artifacts to: /Users/jdipol/tmp/1.4.10/helidon-quickstart-se/target/helidon-quickstart-se.build_artifacts.txt
[WARNING] 	at com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:172)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1426)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
[WARNING] 	at java.base/java.util.concurrent.ForkJoinPool.awaitQuiescence(ForkJoinPool.java:2984)
[WARNING] 	at com.oracle.graal.pointsto.util.CompletionExecutor.complete(CompletionExecutor.java:238)
[WARNING] 	at com.oracle.graal.pointsto.PointsToAnalysis.doTypeflow(PointsToAnalysis.java:659)
[WARNING] 	at com.oracle.graal.pointsto.PointsToAnalysis.finish(PointsToAnalysis.java:634)
[WARNING] 	at com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:704)
[WARNING] 	... 6 more
[WARNING] Error: Image build request failed with exit status 1
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

## Steps to reproduce
```
$ export GRAALVM_HOME=/usr/local/graalvm-ce-java11-21.3.0/Contents/Home
$ mvn archetype:generate -DinteractiveMode=false \
    -DarchetypeGroupId=io.helidon.archetypes \
    -DarchetypeArtifactId=helidon-quickstart-se \
    -DarchetypeVersion=1.4.10 \
    -DgroupId=io.helidon.examples -DartifactId=helidon-quickstart-se \
    -Dpackage=io.helidon.examples.quickstart.se

$ cd helidon-quickstart-se
$ mvn clean install -Pnative-image
```

--- a/grpc/server/src/main/java/io/helidon/grpc/server/SSLContextBuilder.java
+++ b/grpc/server/src/main/java/io/helidon/grpc/server/SSLContextBuilder.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019, 2022 Oracle and/or its affiliates.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -19,6 +19,7 @@ package io.helidon.grpc.server;
 import java.io.IOException;
 import java.security.GeneralSecurityException;
 import java.security.KeyStore;
+import java.security.SecureRandom;
 import java.security.Security;
 import java.security.cert.Certificate;
 import java.security.cert.X509Certificate;
@@ -34,6 +35,7 @@ import javax.net.ssl.TrustManagerFactory;
 
 import io.helidon.common.Builder;
 import io.helidon.common.CollectionsHelper;
+import io.helidon.common.LazyValue;
 import io.helidon.common.pki.KeyConfig;
 import io.helidon.config.Config;
 
@@ -43,7 +45,9 @@ import io.helidon.config.Config;
 public final class SSLContextBuilder implements Builder<SSLContext> {
 
     private static final String PROTOCOL = "TLS";
-    private static final Random RANDOM = new Random();
+    // secure random cannot be stored in native image, it must
+    // be initialized at runtime
+    private static final LazyValue<Random> RANDOM = LazyValue.create(SecureRandom::new);
 
     private KeyConfig privateKeyConfig;
     private KeyConfig trustConfig;
@@ -165,7 +169,7 @@ public final class SSLContextBuilder implements Builder<SSLContext> {
         }
 
         byte[] passwordBytes = new byte[64];
-        RANDOM.nextBytes(passwordBytes);
+        RANDOM.get().nextBytes(passwordBytes);
         char[] password = Base64.getEncoder().encodeToString(passwordBytes).toCharArray();
 
         KeyStore ks = KeyStore.getInstance("JKS");

--- a/webserver/webserver/src/main/java/io/helidon/webserver/SSLContextBuilder.java
+++ b/webserver/webserver/src/main/java/io/helidon/webserver/SSLContextBuilder.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017, 2022 Oracle and/or its affiliates.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
@@ -19,6 +19,7 @@ package io.helidon.webserver;
 import java.io.IOException;
 import java.security.GeneralSecurityException;
 import java.security.KeyStore;
+import java.security.SecureRandom;
 import java.security.Security;
 import java.security.cert.Certificate;
 import java.security.cert.X509Certificate;
@@ -34,6 +35,7 @@ import javax.net.ssl.TrustManagerFactory;
 
 import io.helidon.common.Builder;
 import io.helidon.common.CollectionsHelper;
+import io.helidon.common.LazyValue;
 import io.helidon.common.pki.KeyConfig;
 import io.helidon.config.Config;
 
@@ -44,7 +46,9 @@ import io.helidon.config.Config;
 public final class SSLContextBuilder implements Builder<SSLContext> {
 
     private static final String PROTOCOL = "TLS";
-    private static final Random RANDOM = new Random();
+    // secure random cannot be stored in native image, it must
+    // be initialized at runtime
+    private static final LazyValue<Random> RANDOM = LazyValue.create(SecureRandom::new);
 
     private KeyConfig privateKeyConfig;
     private KeyConfig trustConfig;
@@ -166,7 +170,7 @@ public final class SSLContextBuilder implements Builder<SSLContext> {
         }
 
         byte[] passwordBytes = new byte[64];
-        RANDOM.nextBytes(passwordBytes);
+        RANDOM.get().nextBytes(passwordBytes);
         char[] password = Base64.getEncoder().encodeToString(passwordBytes).toCharArray();
 
         KeyStore ks = KeyStore.getInstance("JKS");

--- a/webserver/webserver/src/main/resources/META-INF/native-image/native-image.properties
+++ b/webserver/webserver/src/main/resources/META-INF/native-image/native-image.properties
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2019, 2022 Oracle and/or its affiliates.
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
@@ -22,4 +22,38 @@ Args=--report-unsupported-elements-at-runtime \
      --allow-incomplete-classpath \
      -H:ReflectionConfigurationResources=${.}/helidon-webserver-reflection-config.json \
      --no-fallback \
-     --initialize-at-run-time=io.netty.buffer.UnpooledByteBufAllocator$InstrumentedUnpooledUnsafeNoCleanerDirectByteBuf,io.netty.buffer.UnreleasableByteBuf,io.netty.handler.codec.http2.Http2ConnectionHandler,io.netty.handler.codec.http2.Http2CodecUtil,io.netty.handler.codec.http.HttpObjectEncoder,io.netty.handler.codec.http2.CleartextHttp2ServerUpgradeHandler,io.netty.handler.codec.http2.DefaultHttp2FrameWriter,io.netty.handler.codec.http2.Http2ServerUpgradeCodec,io.netty.handler.ssl.JdkNpnApplicationProtocolNegotiator,io.netty.handler.ssl.JettyNpnSslEngine,io.netty.handler.ssl.ReferenceCountedOpenSslEngine,io.netty.handler.ssl.ConscryptAlpnSslEngine
+  --initialize-at-run-time=io.netty.buffer.UnpooledByteBufAllocator$InstrumentedUnpooledUnsafeNoCleanerDirectByteBuf \
+  --initialize-at-run-time=io.netty.buffer.UnreleasableByteBuf \
+  --initialize-at-run-time=io.netty.handler.codec.compression.BrotliDecoder$1 \
+  --initialize-at-run-time=io.netty.handler.codec.compression.BrotliOptions \
+  --initialize-at-run-time=io.netty.handler.codec.compression.Lz4FrameDecoder \
+  --initialize-at-run-time=io.netty.handler.codec.compression.Lz4FrameEncoder \
+  --initialize-at-run-time=io.netty.handler.codec.compression.LzmaFrameEncoder \
+  --initialize-at-run-time=io.netty.handler.codec.compression.ZstdOptions \
+  --initialize-at-run-time=io.netty.handler.codec.http.HttpObjectEncoder \
+  --initialize-at-run-time=io.netty.handler.codec.http2.CleartextHttp2ServerUpgradeHandler \
+  --initialize-at-run-time=io.netty.handler.codec.http2.DefaultHttp2FrameWriter \
+  --initialize-at-run-time=io.netty.handler.codec.http2.Http2CodecUtil \
+  --initialize-at-run-time=io.netty.handler.codec.http2.Http2ConnectionHandler \
+  --initialize-at-run-time=io.netty.handler.codec.http2.Http2ServerUpgradeCodec \
+  --initialize-at-run-time=io.netty.handler.codec.marshalling.CompatibleMarshallingDecoder \
+  --initialize-at-run-time=io.netty.handler.codec.marshalling.CompatibleMarshallingEncoder \
+  --initialize-at-run-time=io.netty.handler.codec.marshalling.MarshallingDecoder \
+  --initialize-at-run-time=io.netty.handler.codec.marshalling.MarshallingEncoder \
+  --initialize-at-run-time=io.netty.handler.codec.protobuf.ProtobufDecoder \
+  --initialize-at-run-time=io.netty.handler.ssl.ConscryptAlpnSslEngine \
+  --initialize-at-run-time=io.netty.handler.ssl.JdkNpnApplicationProtocolNegotiator \
+  --initialize-at-run-time=io.netty.handler.ssl.JettyAlpnSslEngine$ClientEngine \
+  --initialize-at-run-time=io.netty.handler.ssl.JettyAlpnSslEngine$ServerEngine \
+  --initialize-at-run-time=io.netty.handler.ssl.JettyNpnSslEngine \
+  --initialize-at-run-time=io.netty.handler.ssl.OpenSslAsyncPrivateKeyMethod \
+  --initialize-at-run-time=io.netty.handler.ssl.OpenSslPrivateKeyMethod \
+  --initialize-at-run-time=io.netty.handler.ssl.ReferenceCountedOpenSslContext \
+  --initialize-at-run-time=io.netty.handler.ssl.ReferenceCountedOpenSslEngine \
+  --initialize-at-run-time=io.netty.internal.tcnative.AsyncSSLPrivateKeyMethod \
+  --initialize-at-run-time=io.netty.internal.tcnative.CertificateVerifier \
+  --initialize-at-run-time=io.netty.internal.tcnative.SSL \
+  --initialize-at-run-time=io.netty.internal.tcnative.SSLPrivateKeyMethod \
+  --initialize-at-run-time=io.netty.util.internal.logging.Log4JLogger \
+  --initialize-at-run-time=io.netty.util.internal.logging.Slf4JLoggerFactory \
+  --initialize-at-run-time=io.netty.handler.ssl.BouncyCastleAlpnSslUtils

